### PR TITLE
@types/node-telegram-bot-api: New type definition

### PR DESCRIPTION
**Pull request by AnotiaWang**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<[url here](https://core.telegram.org/bots/api#december-7-2021)>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/node-telegram-bot-api/index.d.ts [hunk 1]
```diff
@@ -422,6 +429,9 @@ declare namespace TelegramBot {
         permissions?: ChatPermissions | undefined;
         can_set_sticker_set?: boolean | undefined;
         sticker_set_name?: string | undefined;
+        has_private_forwards?: boolean | undefined;
+        has_protected_content?: boolean | undefined;
+        is_automatic_forward?: boolean | undefined;
```
Comment (toniop99) [inline types/node-telegram-bot-api/index.d.ts:434]: I think this prop should't be on the Chat interface, just on the Message interface. Can you share more information?

**Review by andezdev — CHANGES_REQUESTED** [review]
Hi!

It seem all good for me except the property is automatic forward on the Chat interface. Could you edit that or pass information about the prop?

File: types/node-telegram-bot-api/index.d.ts (hunk 1 above)
Comment (AnotiaWang) [reply types/node-telegram-bot-api/index.d.ts]: Sorry for the mistake ;) 
Besides, I added  `slow_mode_delay`, `message_auto_delete_time` and `linked_chat_id` to the Chat interface. API document is [here](https://core.telegram.org/bots/api#chat). Please review them, thanks :)

**Review by andezdev — APPROVED** [review]

**Comment by toniop99** [comment]
Redy to merge

**Comment by toniop99** [comment]
Ready to merge

**Pull request merged**